lib/index.js

```diff
@@ -280,7 +280,7 @@ Metalsmith.prototype.readFile = unyield(function*(file){
 
     if (frontmatter && utf8(buffer)) {
       try {
-        parsed = matter(buffer.toString());
+        parsed = matter(buffer.toString().replace(/\r\n/g, '\n'));
```

Comment:
Yep, this should probably be changed since it's effectively altering the value of the `contents` property further down. Good catch!

Maybe you can kill two birds with one stone here by getting Windows support added upstream in grey-matter.